$NetBSD: patch-ab,v 1.5 2002/03/05 18:29:49 dmcmahill Exp $

Obey CC and CFLAGS

Don't do additional checksums of the source (gets thrown 
off by our patches, plus we already have files/md5)

--- src/makefile.orig	Wed Feb 25 08:18:04 1998
+++ src/makefile	Tue Mar  5 13:09:23 2002
@@ -2,6 +2,6 @@
 
 .SUFFIXES: .c .o
-CC = cc
-CFLAGS = -O
+#CC = cc
+#CFLAGS = -O2
 SHELL = /bin/sh
 YACC = yacc
@@ -25,5 +25,5 @@
 OBJECTS = $(OBJECTSd) $(MALLOC)
 
-all: xsum.out f2c
+all: f2c
 
 f2c: $(OBJECTS)
